nx-X11/lib/modules/im/ximcp/Makefile.am

```diff
@@ -0,0 +1,56 @@
+NULL =
+
+noinst_LTLIBRARIES=libximcp.la
+
+AM_CPPFLAGS =					\
+	-I$(top_srcdir)/include			\
+	-I$(top_srcdir)/include/X11		\
+	-I$(top_builddir)/include 		\
+	-I$(top_builddir)/include/X11		\
+	-I$(top_srcdir)/src/xcms		\
+	-I$(top_srcdir)/src/xkb			\
+	-I$(top_srcdir)/src/xlibi18n		\
+	-I$(top_srcdir)/src			\
+	-I$(top_srcdir)/../exports/include	\
+	-D_GNU_SOURCE				\
```

Comment:
So far, we haven't used either macro. Unless there's a specific reason why `_BSD_SOURCE` would not work (e.g., due to nomachine changes that won't work otherwise), we should keep it in sync with upstream.